jujugui/static/gui/src/app/app.js

```diff
@@ -3023,7 +3036,8 @@ YUI.add('juju-gui', function(Y) {
       // Update the breadcrumb with the new model name.
       this._renderBreadcrumb();
       // Update the page title.
-      document.title = `${environmentName} - Juju GUI`;
+      this.defaultPageTitle = `${environmentName} - Juju GUI`;
```

Comment:
I don't think this is necessary. the setPageTitle should be called with the new value